after a net error the list activity crash by AndroidRuntime: FATAL

In player  Activity
failed to connect to tube.4aem.com/165.227.5.95 (port 443) from /192.168.1.4 (port 46724) after 10000ms
then back to list Activity
 java.lang.RuntimeException: Unable to stop service


`
2018-12-31 13:48:47.523 13973-13973/net.schueller.peertube V/VideoPlayActivity: click: 77de5919-2ea4-4aa9-85bc-c35601bf56fa
2018-12-31 13:48:57.549 13973-13973/net.schueller.peertube E/VideoPlayActivity: failed to connect to tube.4aem.com/165.227.5.95 (port 443) from /192.168.1.4 (port 46724) after 10000ms
    java.net.SocketTimeoutException: failed to connect to tube.4aem.com/165.227.5.95 (port 443) from /192.168.1.4 (port 46724) after 10000ms
        at net.schueller.peertube.activity.VideoPlayActivity$3.onFailure(VideoPlayActivity.java:370)
        at retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$2.run(ExecutorCallAdapterFactory.java:80)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6806)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
2018-12-31 13:48:57.598 13973-13973/net.schueller.peertube I/Toast: Show toast from OpPackageName:net.schueller.peertube, PackageName:net.schueller.peertube
2018-12-31 13:48:58.623 13973-13973/net.schueller.peertube V/VideoPlayActivity: onPause()...
2018-12-31 13:49:00.975 13973-13973/net.schueller.peertube V/VideoPlayActivity: onResume()...
2018-12-31 13:49:34.242 13973-13973/net.schueller.peertube V/VideoPlayActivity: onPause()...
2018-12-31 13:49:34.681 13973-13973/net.schueller.peertube V/VideoPlayActivity: onStop()...
2018-12-31 13:49:34.695 13973-13973/net.schueller.peertube V/VideoPlayerService: onDestroy...
2018-12-31 13:49:34.695 13973-13973/net.schueller.peertube D/AndroidRuntime: Shutting down VM
2018-12-31 13:49:34.697 13973-13973/net.schueller.peertube E/AndroidRuntime: FATAL EXCEPTION: main
    Process: net.schueller.peertube, PID: 13973
    java.lang.RuntimeException: Unable to stop service net.schueller.peertube.service.VideoPlayerService@49e67e7: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.android.exoplayer2.ui.PlayerNotificationManager.setPlayer(com.google.android.exoplayer2.Player)' on a null object reference
        at android.app.ActivityThread.handleStopService(ActivityThread.java:3757)
        at android.app.ActivityThread.access$1800(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1709)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6806)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.android.exoplayer2.ui.PlayerNotificationManager.setPlayer(com.google.android.exoplayer2.Player)' on a null object reference
        at net.schueller.peertube.service.VideoPlayerService.onDestroy(VideoPlayerService.java:122)
        at android.app.ActivityThread.handleStopService(ActivityThread.java:3733)
        at android.app.ActivityThread.access$1800(ActivityThread.java:200) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1709) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6806) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873) 
2018-12-31 13:49:34.720 13973-13973/net.schueller.peertube I/Process: Sending signal. PID: 13973 SIG: 9

`

--- a/app/src/main/java/net/schueller/peertube/service/VideoPlayerService.java
+++ b/app/src/main/java/net/schueller/peertube/service/VideoPlayerService.java
@@ -118,11 +118,13 @@ public class VideoPlayerService extends Service {
     public void onDestroy() {
 
         Log.v(TAG, "onDestroy...");
-
-        playerNotificationManager.setPlayer(null);
-        player.release();
-        player = null;
-
+        if (playerNotificationManager != null) {
+            playerNotificationManager.setPlayer(null);
+        }
+        if (player != null) {
+            player.release();
+            player = null;
+        }
         super.onDestroy();
     }
 

